src/components/Filters/CharactersFilter.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { MouseEventHandler, useState } from 'react';
+import styles from './CharactersFilter.module.scss'
+import arrow from '../../assets/icons/arrow.png';
+
+interface CharactersFilter {
+    statusFilter: string,
+    speciesFilter: string,
+    genderFilter: string,
+    setStatusFilter: (filter: string) => () => void,
+    setSpeciesFilter: (filter: string) => () => void,
+    setGenderFilter: (filter: string) => () => void
+}
+
+const CharactersFilter = (props: CharactersFilter) => {
```

Comment:
Ai conflict de nume intre componenta si tipul props-urilor acesteia. Numele tipului poate fi redenumit in CharactersFilterProps pentru a evita conflictul.